src/Umbraco.Web/PropertyEditors/GridPropertyEditor.cs

```diff
@@ -69,6 +77,15 @@ public override object FromEditor(ContentPropertyData editorValue, object curren
                 if (editorValue.Value == null)
                     return null;
 
+                var config = editorValue.DataTypeConfiguration as GridConfiguration;
+                var mediaParent = config?.MediaParentId;
+                Guid mediaParentId;
+
+                if (mediaParent == null)
+                    mediaParentId = Guid.Empty;
+                else
+                    mediaParentId = mediaParent.Guid;
+
```

Comment:
These five lines could be replaced by 
```
var mediaParentId = mediaParent == null ? Guid.Empty : mediaParent.Guid;
```

Reply:
My brain was in a super verbose mode that day 😃